### PR TITLE
Torch-TensorRT 1.1.0 

### PR DESCRIPTION
**Pull request by narendasan**
# Description

Release PR for Torch-TensorRT 1.1.0

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: core/lowering/passes/linear_to_addmm.cpp [hunk 1]
```diff
@@ -1,15 +1,15 @@
 
 #include <torch/csrc/jit/runtime/operator.h>
+#include "core/util/prelude.h"
```
Comment (andi4191) [inline core/lowering/passes/linear_to_addmm.cpp:3]: Is this change related to linter?

File: tests/util/util.h [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #pragma once
 
+#include <ATen/ATen.h>
 #include <string>
 #include <vector>
-#include <ATen/ATen.h>
```
Comment (andi4191) [inline tests/util/util.h:5]: Is this also related to linter?

**Review by andi4191 — APPROVED** [review]
A couple of questions. Rest LGTM

File: core/lowering/passes/linear_to_addmm.cpp (hunk 1 above)
Comment (narendasan) [reply core/lowering/passes/linear_to_addmm.cpp:3]: yeah, a bunch of people were failing the linter so im fixing it all at once

File: tests/util/util.h (hunk 2 above)
Comment (narendasan) [reply tests/util/util.h:5]: yes
